service/application/src/main/resources/application.yml

```diff
@@ -88,164 +88,190 @@ eventPortal:
-#      connections:
-#        - name: myConfluentTLS
-#          url: ${CONFLUENT_SCHEMA_REGISTRY_URL}
-#          authentication:
-#            - protocol: TLS
-#              credentials:
-#                - properties:
-#                    - name: ssl.truststore.location
-#                      value: ${TRUSTSTORE_LOCATION}
-#                    - name: ssl.truststore.password
-#                      value: ${TRUSTSTORE_PASSWORD}
-#                  source: ENVIRONMENT_VARIABLE
-#                  operations:
-#                    - name: ALL
-#    # Confluent Schema Registry MTLS example
-#    - id: someConfluentId
-#      name: no auth confluent example
-#      type: confluent_schema_registry
-#      relatedServices:
-#        - id1
-#      connections:
-#        - name: myConfluentMTLS
-#          url: ${CONFLUENT_SCHEMA_REGISTRY_URL}
-#          authentication:
-#            - protocol: MTLS
-#              credentials:
-#                - properties:
-#                    - name: ssl.truststore.location
-#                      value: ${TRUSTSTORE_LOCATION}
-#                    - name: ssl.truststore.password
-#                      value: ${TRUSTSTORE_PASSWORD}
-#                    - name: ssl.keystore.password
-#                      value: ${KEYSTORE_PASSWORD}
-#                    - name: ssl.keystore.location
-#                      value: ${KEYSTORE_LOCATION}
-#                    - name: ssl.key.password
-#                      value: ${KEY_PASSWORD}
-#                  source: ENVIRONMENT_VARIABLE
-#                  operations:
-#                    - name: ALL
-#    # Solace example
-#    - id: solaceDefaultService
-#      type: SOLACE
-#      name: staging service
-#      connections:
-#        - name: mySolaceServiceSemp
-#          url: ${SOLACE_SEMP_URL}
-#          authentication:
-#            - protocol: SEMP
-#              credentials:
-#                - source: ENVIRONMENT_VARIABLE
-#                  operations:
-#                    - name: ALL
-#                  properties:
-#                    - name: username
-#                      value: ${SOLACE_SEMP_USERNAME}
-#                    - name: password
-#                      value: ${SOLACE_SEMP_PASSWORD}
-#          properties:
-#            - name: msgVpn
-#              value: ${SOLACE_VPN_NAME}
-#            - name: sempPageSize
-#              value: 100
+plugins:
+  resources:
+    # IBMMQ example
+    - id: ibmmqDefaultService
+      type: IBMMQ
+      name: test service
+      connections:
+        - name: ibmmqCloudService
+          url: https://web-qm1-5bc5.qm.us-south.mq.appdomain.cloud/ibmmq/rest/v1/admin/qmgr/QM1
+          authentication:
+            - protocol: BASIC
+              credentials:
+                - source:
+                  properties:
+                    - name: username
+                      value: ush
+                    - name: password
+                      value: E5J8yGHjPTErikzOVZ0FiloP47FooNKQSTcTx5jQE-oM
```

Comment:
Can these examples use ENV vars like `${SOME_USERNAME}` since that's the preferred way?